lib/database/models/triggerCounters.js

```diff
@@ -0,0 +1,62 @@
+const Sequelize = require('sequelize');
+
+module.exports = (sequelize) => {
+    const TriggerCounters = sequelize.define('TriggerCounters', {
+        id: {
+            allowNull: false,
+            primaryKey: true,
+            autoIncrement: true,
+            type: Sequelize.INTEGER,
+        },
+        timestamp: {
+            allowNull: false,
+            type: Sequelize.DATE,
+        },
+        runNumber: {
+            allowNull: false,
+            type: Sequelize.INTEGER,
+        },
+        className: {
+            allowNull: false,
+            type: Sequelize.STRING,
+        },
+        // LM Before
+        lmb: {
+            allowNull: false,
+            type: Sequelize.BIGINT,
+            unsigned: true,
+        },
+        // LM After
+        lma: {
+            allowNull: false,
+            type: Sequelize.BIGINT,
+            unsigned: true,
+        },
+        l0b: {
+            allowNull: false,
+            type: Sequelize.BIGINT,
+            unsigned: true,
+        },
+        l0a: {
+            allowNull: false,
+            type: Sequelize.BIGINT,
+            unsigned: true,
+        },
+        l1b: {
+            allowNull: false,
+            type: Sequelize.BIGINT,
+            unsigned: true,
+        },
+        l1a: {
+            allowNull: false,
+            type: Sequelize.BIGINT,
+            unsigned: true,
+        },
+    }, { indexes: [{ unique: true, fields: ['id', 'className'] }] });
```

Comment:
Should it be `id` or `run_number` as in `lib/database/migrations/20240603082112-create-trigger-counters.js` ?

Reply:
Indeed, fixed